app/styles/app.scss

```diff
@@ -239,3 +239,5 @@ body {
 .tt-suggestion:not(:hover) .inactive {
   background-color: rgb(230, 230, 230);
 }
+
+@import "ember-power-select";
```

Comment:
why is this import not at the top?

Reply:
This is duplicated.. The import is in `app/styles/ember-power-select-custom.scss`.

Reply:
Fixed!